src/pages/admin/profile/[user_id].tsx

```diff
@@ -0,0 +1,175 @@
+import { Suspense, useContext, useEffect, useState } from "react";
+
+import { NextRouter, useRouter } from "next/router";
+import { userContext } from "supabase/user_context";
+import { useSupabaseClient } from "@supabase/auth-helpers-react";
+import { Database } from "supabase/db_types";
+
+import { Avatar, Chip, Typography, Stack, Button, Box, TextField, FormHelperText } from "@mui/material";
+import MaleIcon from "@mui/icons-material/Male";
+import FemaleIcon from "@mui/icons-material/Female";
+import TransgenderIcon from "@mui/icons-material/Transgender";
+import CakeIcon from "@mui/icons-material/Cake";
+
+import Navbar from "@/components/admin/AdminNavbar";
+import Loading from "@/components/public/Loading";
+
+import { User } from "@/types/User";
+import { PagePaths } from "enum/pages";
+import { Gender } from "enum/gender";
+import { CHAR_LIMIT } from "enum/inputLimit";
+import { GetUserByUserId, UpdateUserNationalIdByUserId } from "@/services/User";
+import AdminVerifyDialog from "@/components/admin/AdminVerifyDialog";
+import { validateNationalIDCardNumber } from "@/utilities/validation";
+import { validation } from "@/types/Validation";
+import VerifyChip from "@/components/profile/VerifyChip";
+
+// style
+const profile_layout = {
+  width: "30vw",
+  minWidth: "200px",
+};
+const avatar = { width: 200, height: 200 };
+
+export default function adminProfile() {
+  const router: NextRouter = useRouter();
+  const userStatus = useContext(userContext);
+  const supabaseClient = useSupabaseClient<Database>();
+
+  const [targetUserData, setTargetUserData] = useState<User | null>(null);
+  const [nationalIDCard, setnationalIDCard] = useState<string>("");
+
+  const [isVerifyModalShow, setIsVerifyModalShow] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
+  const [errMsg, setErrMsg] = useState<string>("");
```

Comment:
Should this one be included in AdminVerifyDialog or here?

Reply:
I have already refactor this point (And the other 2 points like this)

https://github.com/2110336-2565-2/sec3-group13-playhub/pull/7/commits/53608a3a21e616d12f024113825d389113baeded